tests/ui/components/results/verticalresultscountcomponent.js

```diff
@@ -42,17 +42,8 @@ describe('results count component', () => {
   });
 
   it('works with search offset', () => {
-    const COMPONENT_MANAGER = mockManager({
-      globalStorage: {
-        on: () => {},
-        getState: key => {
-          if (key === StorageKeys.SEARCH_OFFSET) {
-            return 40;
-          }
-          return null;
-        }
-      }
-    });
+    const COMPONENT_MANAGER = mockManager();
+    COMPONENT_MANAGER.core.storage.set(StorageKeys.SEARCH_OFFSET, 40);
```

Comment:
We return `undefined` instead of `null` (as was done previously) in the storage if the key doesn't exist. I don't think this matters since tests are passing, just pointing it out